### PR TITLE
Just disable api keys

### PR DESCRIPTION
**Pull request by dmannarino**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Make sure you are requesting to pull a topic/feature/bugfix branch (right side). Don't request your master!
- [x] Make sure you are making a pull request against the develop branch (left side). Also you should start your branch off our develop.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Other (please describe): 

Disable API keys in staging to unblock Vizz. on TCLF work

## What is the current behavior?

- API keys are required for download, analysis, query endpoints

Issue Number: N/A

## What is the new behavior?

- API keys are NOT required for download, analysis, query endpoints

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

The migrations stuff is just to get tests to pass in dev. Will revert before merging.

### CODE REVIEW

File: app/models/orm/migrations/env.py [hunk 1]
```diff
@@ -98,7 +98,7 @@ def run_migrations_online():
         with context.begin_transaction() as transaction:
             context.run_migrations()
             if "dry-run" in context.get_x_argument():
-                print("Dry-run succeeded; now rolling back transaction")
+                print("Dry-run succeeded; now rolling back transactionapp")
```
Comment (gtempus) [inline app/models/orm/migrations/env.py:101]: i know this isn't part of your PR changes, @dmannarino, but is this a typo?

**Review by solomon-negusse — APPROVED** [review]

File: app/models/orm/migrations/env.py (hunk 1 above)
Comment (tanderegg) [inline app/models/orm/migrations/env.py:101]: Was this intentional?  I'm assuming it should still just be "now rolling back transaction", but I could certainly be missing something!

**Review by tanderegg — APPROVED** [review]
Looks good, with one minor (possible) typo in a string.

File: app/models/orm/migrations/env.py (hunk 1 above)
Comment (dmannarino) [reply app/models/orm/migrations/env.py:101]: It is. I didn't bother to fix it as I will be reverting the commit anyway.

Comment (dmannarino) [reply app/models/orm/migrations/env.py:101]: Not intentional, but included in a commit cherry-picked from another PR. I didn't bother to fix it as I will be reverting the commit before merge anyway.

File: app/routes/analysis/analysis.py [hunk 2]
```diff
@@ -80,7 +81,8 @@ async def zonal_statistics_get(
     deprecated=True,
 )
 async def zonal_statistics_post(
-    request: ZonalAnalysisRequestIn, api_key: APIKey = Depends(get_api_key)
+    request: ZonalAnalysisRequestIn,
+    # api_key: APIKey = Depends(get_api_key)
```
[the remark this answers is not in the document]
Comment (dmannarino) [reply app/routes/analysis/analysis.py:85]: So you mean leave the dependency here, but modify get_api_key?

[the remark this answers is not in the document]
Comment (dmannarino) [reply app/routes/analysis/analysis.py:85]: I'll admit that would mean modifying far fewer locations.

**Pull request merged**